src/avk.cpp

```diff
@@ -2,6 +2,8 @@
 #include <avk/avk_log.hpp>
 #include <avk/avk.hpp>
 
+#define _SILENCE_CXX20_CISCO646_REMOVED_WARNING

```

Comment:
What's this and is this really a good idea?

Reply:
✅